File: javascript/7_kyu/cw_sumofsequence.js
const sequenceSum = (begin, end, step) => {
  var answer = 0;
  if (begin > end) {
   return 0;
   }
   for (i = begin; i <= end; i += step) {
     answer += i;
   }
   return answer;
};

// https://www.codewars.com/kata/586f6741c66d18c22800010a

File: ruby/7_kyu/cw_sumofsequence.rb
def sequence_sum(begin_number, end_number, step)
  if end_number < begin_number
    return 0
  else
    return (begin_number..end_number).step(step).reduce(:+)
  end
end

# https://www.codewars.com/kata/586f6741c66d18c22800010a